src/google/protobuf/repeated_field_unittest.cc

```diff
@@ -1866,12 +1920,13 @@ TEST_F(RepeatedPtrFieldIteratorTest, ConstReverseIteration) {
   const RepeatedPtrField<std::string>& const_proto_array = proto_array_;
   RepeatedPtrField<std::string>::const_reverse_iterator iter =
       const_proto_array.rbegin();
+  iter - const_proto_array.crbegin();
```

Comment:
This expression is unused.
Was it meant to just see that it compiles?

Reply:
I basically mirrored the `ConstIteration` test that does the same thing
 
https://github.com/protocolbuffers/protobuf/blob/5028da4153fbfe3e1783cfb06d9c5f088fa31e71/src/google/protobuf/repeated_field_unittest.cc#L1844

But I'm just going to remove it in both places